charts/csi-isilon/Chart.yaml

```diff
@@ -2,10 +2,9 @@ apiVersion: v2
 name: csi-isilon
 version: 2.11.0
 appVersion: "2.11.0"
-kubeVersion: ">= 1.21.0 < 1.31.0"
```

Comment:
The minimum kube version isn't consistent across drivers, do you know why that is?